system_parameters_tests.py

```diff
@@ -0,0 +1,1042 @@
+import unittest
+import utils
+import config
+import requests
+import json
+import funcs
+import os
+import time
+
+
+class SystemParametersCase(unittest.TestCase):
+    def setUp(self):
+        global url, token, prKey, pause,\
+            dbHost, dbName, login, password, contractName, maxInt, minInt, defaultValues, defaultValueKey
+        self.config = config.getNodeConfig()
+        url = self.config["2"]["url"]
+        pause = self.config["1"]["time_wait_tx_in_block"]
+        prKey = self.config["1"]['private_key']
+        self.data = utils.login(url, prKey)
+        token = self.data["jwtToken"]
+        dbHost = self.config["2"]["dbHost"]
+        dbName = self.config["2"]["dbName"]
+        login = self.config["2"]["login"]
+        password = self.config["2"]["pass"]
+        contractName = "UpdateSysParam"
+        maxInt = 9223372036854775807
+        minInt = -9223372036854775808
+        defaultValueKey = "system_parameters"
+        defaultValues = self.readDefaultParameters()
+
+    def readDefaultParameters(self):
+        path = os.path.join(os.getcwd(), "defaultSystemParameters.json")
+        with open(path, 'r') as f:
+            data = f.read()
+        return json.loads(data)
+
+    def assertTxInBlock(self, result, jwtToken):
```

Comment:
it repeats in every test case, mayber better create something like TxTestCase,  and then inherit from it